qa/websocket_server.py

```diff
@@ -58,7 +53,25 @@
 """
```

Comment:
pls, update the description of the test.

Reply:
Done